src/screens.native.js

```diff
@@ -57,32 +69,52 @@ export class Screen extends React.Component {
       const { active, onComponentRef, ...props } = this.props;
 
       return <Animated.View {...props} ref={this.setRef} />;
-    } else if (version.minor >= 57) {
-      return <AnimatedNativeScreen {...this.props} ref={this.setRef} />;
     } else {
-      // On RN version below 0.57 we need to wrap screen's children with an
-      // additional View because of a bug fixed in react-native/pull/20658 which
-      // was preventing a view from having both styles and some other props being
-      // "animated" (using Animated native driver)
-      const { style, children, ...rest } = this.props;
-      return (
-        <AnimatedNativeScreen
-          {...rest}
-          ref={this.setRef}
-          style={StyleSheet.absoluteFill}>
-          <Animated.View style={style}>{children}</Animated.View>
-        </AnimatedNativeScreen>
-      );
+      AnimatedNativeScreen =
+        AnimatedNativeScreen ||
+        Animated.createAnimatedComponent(ScreensNativeModules.NativeScreen);
```

Comment:
it is a bad idea to use `createAnimatedComponent` inside render. It's like you'd define a component class in render method and then returned it

Reply:
It's only initialized once. All the subsequent renders will use the already initialized version.